omod-1.9/pom.xml

```diff
@@ -11,7 +11,7 @@
 	<description>OpenMRS module project for Rest Web Services</description>
 
 	<properties>
-		<openmrs.version.1.9>1.9.10</openmrs.version.1.9>
+		<openmrs.version.1.9>1.9.13-SNAPSHOT</openmrs.version.1.9>
```

Comment:
I think this is not necessary anymore since the tests are in the omod-2.0 subpackage.